checklist.md

```diff
@@ -0,0 +1,356 @@
+
+    **Results** 
+    The code does not contain or touch any private information such as Personal Identifiable Information (PII) or Protected Health Information (PHI). 
+
+
+### Code Review
+
+The existing codebase should be given a one time, top-to-bottom code quality and security vulnerability review by two (or more) engineers who have written production code within the past two years, in the languages used in the project Engineers should review credential management practices with the development team to ensure that any keys, passwords, or other sensitive configurations are not checked into source code or in the git history.
+
+The engineers can be federal government employees or trusted partners from outside the agency from other contracts, or from independent testing contracts. Their names, organizations, comments and approval/disapproval on the overall codebase should be tracked in this document.
+
+To provide independent review, ideally the engineers should not have been involved in the development of the software product. This includes engineers who wrote part of the software or who directly provided technical direction and oversight in the creation of the software.
+
+As part of the code review, engineers should reference modern listings of the most significant software security vulnerabilities. For instance, an acceptable description would be that the engineers showed how they used automated tools and manual review to check each item in __OWASP's current 10 Most Critical Web Application Security Risks.__
+
+
+   **Results**
+   The code has been givien a top-to-bottom code quality and security vulnerability review, OWAP's list of current application risks was used for reference. 
+   - Injection - Data cannot be interjected through any form inputs or other data submissions.  
+   - Broken Authentication - App does not store or use credentials. This is a tooling application with automated validation.
+   - Sensitive Data Exposure - Application does not store sensitive data or PII.
+   - XML External Entities (XEE) - Application does not parse or process XML data.
+   - Broken Access Control - Access control is not an issue, application doesn't require user accounts or roles.
+   - Security Misconfiguration - Relies on GitHub actions for automation and testing.
+   - Cross-Site Scripting (XSS) - Application does not have a web based user interface or form for user input.
+   - Insecure Deserialization - Application does not perform deserialization of external or user provided data. </b>
+
+   - The scripts (gen-gource-logs-on-repos.sh and run-scc-on-repos.sh) contain hardcoded empty TOKEN variables. These should be moved to environment variables or a different secure system
```

Comment:
@IsaacMilarky @decause-gov Should we make the error handling + environment var changes as described here or are we good with moving forward?

Reply:
sufficient changes were made over here